JSDemos/Demos/HtmlEditor/Tables/description.md

```diff
@@ -0,0 +1,9 @@
+DevExtreme **HtmlEditor** allows you to create and manage tables. This demo shows how to perform the following operations in the UI:
+
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#inserttable" target="_blank">insertTable</a>
+- <a href="" target="_blank">insertRowAbove</a> / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertrowbelow" target="_blank">insertRowBelow</a>
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertcolumnright" target="_blank">insertColumnRight / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertcolumnleft" target="_blank">insertColumnLeft
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletetable" target="_blank">deleteTable
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deleterow" target="_blank">deleteRow / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletecolumn" target="_blank">deleteColumn
+
+When users click a toolbar [item](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Configuration/toolbar/items/), it calls the method with the same name. You can also call these methods programmatically via the [getModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#getModulemoduleName) method. Refer to the following documentation file to see the full list of supported operations with examples: <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md">DevExtreme Quill modules: Table</a>.
```

Comment:
> it calls the method with the same name

Passive voice seems more suitable here: "a method with the same name is called".

> call these methods **programmatically**

Looks exessive.

>  via the [getModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#getModulemoduleName) method

This is a bit vague. How do methods get called _via_ another method? We should give more details:

"These methods belong to the Table module of Quill. You can use the getModule method to access it and call Table methods at any point in your application."

> the full list of supported operations

"Table methods"?